recipe-server/normandy/recipes/api/v2/views.py

```diff
@@ -23,6 +24,8 @@
     RecipeRevisionSerializer,
     RecipeSerializer,
 )
+import normandy.recipes.api.v2.shield_identicon as shield_identicon
```

Comment:
This is equivalent to a `from` import, since we're not renaming the module: `from normandy.recipes.api.v2 import shield_identicon`. We should use that.

Also, these imports should follow our import style guidelines (that no one probably told you about that we should have): http://mozweb.readthedocs.io/en/latest/reference/python-style.html#import-statements